lib/junk_drawer/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module JunkDrawer
-  VERSION = '1.7.0'
+  VERSION = '1.7.1'
```

Comment:
Nit, this seems like it should be at least a minor change in case anything was expecting previous to be cleared.

Reply:
Okay, I bumped it to `1.8.0`.